modules/core_project_factory/variables.tf

```diff
@@ -147,3 +147,8 @@ variable "disable_dependent_services" {
   type        = string
 }
 
+variable "shared_vpc_enabled" {
+  description = "Existing of shared vpc"
+  type        = bool
+  default     = false
```

Comment:
This should not have a default. We want to ensure that both public modules explicitly pass this argument.

Reply:
Done